### ISSUE TITLE
issue in Xcode 8.1

### ISSUE THREAD

**Issue by yagneshGingercube** [TimOliver/TOCropViewController#135]
Getting the following error when building with simulator, under Xcode 8.1:

**Error: TOCropViewController Resources.bundle: bundle format unrecognized, invalid, or unsuitable Command /usr/bin/codesign failed with exit code 1**

**Comment by TimOliver** [comment]
Um, it's working fine for me! In both the sample app, as well as installed via CocoaPods in one of my apps.

There's no such file as `TOCropViewController Resources.bundle` in this project. Can you please provide some more information on how you've actually tried to integrate this library into your app?

**Comment by codingtomato** [comment]
Hi! I also get the same error when building with simulator in Xcode 8.1 .  But it's OK when I run my app on my phone.   Before Xcode 8.1 all going well. The problem occurred when I just updated my xcode to 8.1 and nothing changed with my project. Bellow is the detailed information provide by Xcode：
<img width="1013" alt="2016-11-02 11 20 27" src="https://cloud.githubusercontent.com/assets/16399091/19915592/81b00a88-a0ee-11e6-803c-90e271137ca4.png">
I guess there my be some problem with using Cocoapods.But I have no idea about where is error.  Here is my Podfile:
…………
…………
…………
…………
…………
pod 'TOCropViewController', '~> 1.1.3'
…………
…………
…………
…………
…………
…………
…………
…………
…………
…………
…………
…………
…………

Thanks a lot!!

**Comment by yagneshGingercube** [comment]
Thanks for quick reply 

I have installed SDK via cocoa pods . I am using latest version of TOCropViewController SDK . I am using it since last year . It was working excellent so far 

Yesterday i installed latest version of XCODE . While compiling with latest xcode , i got an error . I am getting this error only when i am running in iPhone Simulator not on device .

Thanks a lot

**Comment by yagneshGingercube** [comment]
I think you should add directory named 'Resources' in your prototype app and use SDK with Cocoapods to reproduce this issue .

I am not sure but i think renaming 'Resources' directory in your SDK might solve the issue . 

Thanks

**Comment by TimOliver** [comment]
What happened? :)

**Comment by yagneshGingercube** [comment]
I fixed the issue

**Comment by TimOliver** [comment]
@yagneshGingercube Haha okay. Can you please explain what the problem was? Is there anything I can do to avoid it?

@codingtomato Why are you using `'~> 1.1.3'` in your pod file? We're up to v2.0.11 now. :)

**Comment by yagneshGingercube** [comment]
I have deleted localization folders from Resources Group (Removed reference) and Added them back into same "Resources" group . 

![screen shot 2016-11-02 at 3 34 24 pm](https://cloud.githubusercontent.com/assets/9199503/19924432/e2b793aa-a111-11e6-957e-8f9687a44396.png)

![screen shot 2016-11-02 at 3 22 38 pm](https://cloud.githubusercontent.com/assets/9199503/19924117/941fb5b6-a110-11e6-8b16-180228e463b4.png)

Before deleting files , I observed that group icon was shown in blue color instead of default yellow color.  I guessed there can be issue reference with these localization folders. So I decided to remove reference those localization folders and add them back again. And Luckily , it worked . TOCropViewController is working again . I am able to build , run my source code in device as well as iPhone simulator . Also I am able to archive build too . 

Overall Its working excellent for me now . 

Note : Make sure to delete derived data before building project 

:)
